build/templates/publish.yml

```diff
@@ -39,9 +39,6 @@ parameters:
   - name: preRelease
     type: boolean
     default: false
-  - name: noVerify

```

Comment:
What does this `noVerify` skip verifying?

Reply:
apparently '--noVerify' is an option when using `vsce package` not `vsce publish`